app/models/quarter.rb

```diff
@@ -32,7 +32,12 @@ def self.get_date_from(object)
   end
 
   def self.current_quarter
-    new(Time.current.year * 100 + (Time.current.month / 4).ceil * 15)
+    Time.current.month
+    season_hash = Quarter::MONTHS.select { |season, month_range| month_range.include?(Time.current.month) }
```

Comment:
Unused block argument - `season`. If it's necessary, use `_` or `_season` as an argument name to indicate that it won't be used.<br>Line is too long. [107/80]